packages/components/src/internal/components/entities/DataTypeSelector.tsx

```diff
@@ -64,6 +67,109 @@ export const getUncheckedEntityWarning = (
     return null;
 };
 
+interface DataTypeSelectorListProps {
+    columns?: number;
+    dataType?: DataTypeEntity;
+    dataTypes?: DataTypeEntity[];
```

Comment:
Looks like this interface is used as the type for the single item component as well as the multiple item listing component, necessitating the use of optional properties Seems like it would be better to have two separate interfaces with fewer optional props.

Reply:
done